File: thin_client/app/src/main/java/com/example/thin_client/data/model/PrivateProfile.kt
package com.example.thin_client.data.model

import com.example.thin_client.data.Message

data class PrivateProfile (
    val username: String,
    val firstname : String,
    val lastname : String,
    val password : String,
    val avatar : String /* String for the moment eventually needs to be image */,
    val rooms_joined: ArrayList<String>
)

File: fat_client/WPFUI/Models/PrivateProfile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WPFUI.Models
{
    public class PrivateProfile
    {
        public string firstname;
        public string lastname;
        public string username;
        public string password;
        public string avatar;
        public string[] rooms_joined;
        public PrivateProfile(string username, string firstname, string lastname, string password, string avatar
 )
        {
            this.username = username;
            this.firstname = firstname;
            this.lastname = lastname;
            this.password = password;
            this.avatar = avatar;
            this.rooms_joined = new string[50];
        }

    }
}
